code/modules/projectiles/boxes_magazines/internal/rifle.dm

```diff
@@ -28,3 +28,23 @@
 	max_ammo = 1
 	caliber = CALIBER_HARPOON
 	ammo_type = /obj/item/ammo_casing/harpoon
+
+/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow
+	max_ammo = 1
+	caliber = CALIBER_REBAR
+	ammo_type = /obj/item/ammo_casing/rebar
+
+/obj/item/ammo_box/magazine/internal/boltaction/rebarxbowforce
```

Comment:
You can also do subtyping with this. On one hand they share very little between each other, but on the other hand it's good organization.

Reply:
This one im worried we'll get huge file names like /obj/item/ammo_box/magazine/internal/boltaction/rebarxbow/syndie/normal